_includes/lesson.html

```diff
@@ -0,0 +1,27 @@
+<div class='lesson'>
+  <h2>#{{lesson.number}}
+    <small>
+      {{lesson.topic}}
+    </small>
+  </h2>
+
+  <p>{{lesson.description}}</p>
+
+  <ul>
+    {% for slide in lesson.slides %}
+      <li><a href="http://rubymonstas.ch/slides.rubymonstas.ch/slides/{{slide[0]}}/">Slides: {{ slide[1] }}</a> (<a href="https://github.com/rubymonstas-zurich/slides.rubymonstas.ch/releases/latest/download/{{slide[0]}}.pdf">PDF</a>)</li>
+    {% endfor %}
+  </ul>
+
+  <ul>
+    {% for exercise in lesson.exercises %}
+      <li><a href="/exercises/{{ lesson.number }}/{{ exercise[0] }}">Exercises: {{ exercise[1] }}</a></li>
```

Comment:
What do you think of using headings for slides, exercises and additional materials? (I know it requires more conditionals)

Reply:
yeah, i thought about it, but i think it's just going to be too many headings on this page. it's not really obvious what belongs to what anymore... maybe we can experiment a bit more and find something that works...